mopidy_local/storage.py

```diff
@@ -182,12 +178,10 @@ def _cleanup_images(self):
         logger.info("Cleaning up image directory")
         with self._connect() as c:
             uris = set(schema.get_image_uris(c))
-        for root, _, files in os.walk(self._image_dir):
-            for name in files:
-                if uritools.urijoin(self._base_uri, name) not in uris:
-                    path = os.path.join(root, name)
-                    logger.info("Deleting file %s", path)
-                    os.remove(path)
+        for image_path in self._image_dir.glob("**/*"):
```

Comment:
> Note Using the “**” pattern in large directory trees may consume an inordinate amount of time.

Should we be using `glob.iglob` or are we going to have this problem anyway. Same would go for any other uses of glob.

Reply:
I guess that note mostly refers to "if you run this on something unknown, you don't know how long it will take". We only use it on dirs we've built ourselves. The size of this directory tree isn't any larger than the number of albums in your local music collection.